src/ledger/accountobjects.ts

```diff
@@ -0,0 +1,21 @@
+import {removeUndefined} from '../common'
+import {RippleAPI} from '../api'
+import {AccountObjectsResponse} from '../common/types/commands/account_objects'
+
+export default async function getAccountObjects(
+  this: RippleAPI, address: string, options: any = {}
+): Promise<AccountObjectsResponse> {
+  // Intentionally omit local validation.
+  // Validation is performed by rippled.
```

Comment:
Do you want to validate the `options` object as we do in other functions? I'm okay if we don't, but I'd clarify the comment since we're talking about verifying the `options`/`GetAccountObjectsOptions` object, which is owned by ripple-lib and is never sent directly to rippled.